starter/script.js

```diff
@@ -1 +1,58 @@
 // put code here!
+var name = prompt ("What is your name?");
+alert ("Hello," + " " + name + "." + " You are a professional chef and need to prepare a meal for Gordon Ramsey.");
+
+meal ();
+
+function meal () {
+  var mealtype = prompt ("Are you going to make breakfast, lunch, dinner, or dessert?");
```

Comment:
Instead of naming this variable mealtype, make sure to use JS convention of camelCase. I would instead call it `var mealType`. Not a big deal here, but it's good to get in the habit of using the convention!